src/components/combat/Combat.module.scss

```
@@ -0,0 +1,107 @@
+@import '/src/styles/common.scss';
+
+
+                
+$darkBlue: #1f2343;
+
+@mixin mobile {
+  @media (max-width: 786px) {
+    @content;
+    flex-direction: column;
+  }
+}
+
+.wrapper {
+  display: flex;
+  align-items: center;
+  font-family: 'Proxima Nova Alt Lt'; 
+  padding: 50px; 
+  width: 100%;                                             
+  @include mobile();
+}
+
+.left {
+  width: 50%;
+  padding-right: 100px;
+
+  h2 {
+    font-size: 46px;
+    color: $darkBlue;
+    margin-bottom: 20px;
+    font-weight: 900;
+
+    @include mobile() {
+     font-size: 32px;
+    }
+  }
+
+  p {
+    color: #585b71;
+    margin-bottom: 20px;
+    line-height: 25px;
+  }
+
+  @include mobile() {
+    padding-right: 0;
+    padding-bottom: 20px;
+    width: 100%;
+  }
+}
+
+.right {
+  width: 50%;
+  padding-left: 40px;
+  @include mobile() {
+    width: 100%;
+    padding-left: 0;
+  }
+}
+
+.grid {
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  column-gap: 8px;
+  row-gap: 12px;
+  div {
```

It's better to have a blank line between every selector for readability purposes, and please make sure to update it at all places. 

```suggestion
  row-gap: 12px;
  
  div {
```